src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -53,7 +47,6 @@ You can find the `configs` file in your content drive by clicking on the mountpo
     ['#', 'Key', 'Value'],
     ['1', 'commerce-endpoint', 'https://catalog-service.adobe.io/graphql'],
     ['2', 'commerce-core-endpoint', 'https://mystorefront.com/graphql'],
-    ['3', 'commerce-root-category-id', '2'],
```

Comment:
I thought this remains for now?

Reply:
It's so out of place but sure. I put it back.

Reply:
per I talk. I removed it again.